### PR TITLE
docs: add link to fork repo how-to

### PR DESCRIPTION
**Pull request by yallen011**
add a link from github doc on how to fork a repo. 

this will enhance user experience for users new to github by putting the information on how to fork a repo at their finger tips without them having to do additional work to search for it.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ Before you submit your Pull Request (PR) consider the following guidelines:
    We cannot accept code without a signed CLA.
    Make sure you author all contributed Git commits with email address associated with your CLA signature.
 
-4. Fork the angular/angular repo.
+4. [Fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) the angular/angular repo.
 
 5. Make your changes in a new git branch:
```
Comment (aikidave) [inline CONTRIBUTING.md:94]:
```suggestion
5. In your forked repository, make your changes in a new git branch:
```

**Review by aikithoughts — APPROVED** [review]
One suggestion, but nothing blocking.

Reviewed-for: global-docs-approvers

**Comment by yallen011** [comment]
commited change @aikidave suggested

**Review by josephperrott — APPROVED** [review]
LGTM

**Pull request closed without merging**